### ISSUE TITLE
Unable to fetch ami based on existing criteria

### ISSUE THREAD

**Issue by Ayush-Singhal28** [terraform-aws-modules/terraform-aws-autoscaling#104]
Trying to create asg using example mentioned in repo. At **terraform plan**  . Unable to get ami
```
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.

data.aws_vpc.default: Refreshing state...
data.aws_ami.amazon_linux: Refreshing state...
data.aws_security_group.default: Refreshing state...
data.aws_subnet_ids.all: Refreshing state...

```
Error: Your query returned no results. Please change your search criteria and try again.

  on main.tf line 28, in data "aws_ami" "amazon_linux":
  28: data "aws_ami" "amazon_linux" {

```

```

**Comment by Ayush-Singhal28** [comment]
https://github.com/terraform-aws-modules/terraform-aws-autoscaling/pull/105

**Comment by bryantbiggs** [comment]
closed with #105
